src/main/java/frc/robot/RobotContainer.java

```diff
@@ -75,53 +84,74 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
-    swerveDrive.setDefaultCommand(new SwerveDriveCommand(
-        swerveDrive,
-        driver,
-        XboxController.Axis.kLeftY.value,
-        XboxController.Axis.kLeftX.value,
-        XboxController.Axis.kRightX.value,
-        true,
-        false));
-
-    new JoystickButton(driver, XboxController.Button.kLeftBumper.value)
-        .whenPressed(new ParallelCommandGroup(new InstantCommand(mIntake::runIntake, mIntake),
-            new InstantCommand(mIndexer::setIntake, mIndexer)))
-        .whenReleased(new SequentialCommandGroup(new InstantCommand(mIntake::stopIntake, mIntake), new WaitCommand(0.5),
-            new InstantCommand(mIndexer::setIdle, mIndexer)));
-
-    new JoystickButton(driver, XboxController.Button.kX.value)
-        .whenPressed(new InstantCommand(mIntake::ejectIntake, mIntake))
-        .whenReleased(new InstantCommand(mIntake::stopIntake, mIntake));
-
-    new JoystickButton(op, XboxController.Button.kA.value)
-        .whenPressed(new InstantCommand(swerveDrive::zeroGyro));
-
-    new JoystickButton(driver, XboxController.Button.kRightBumper.value)
-        .whenPressed(new TeleopShoot(driver,
-            XboxController.Axis.kLeftY.value,
-            XboxController.Axis.kLeftX.value,
-            XboxController.Axis.kRightX.value,
+    swerve.setDefaultCommand(new SwerveDriveCommand(
+            swerve,
+            driver,
+            kOI.DRIVE_TRANSLATION_Y,
+            kOI.DRIVE_TRANSLATION_X,
+            kOI.DRIVE_ROTATE,
             true,
-            false));
-    //
+            false
```

Comment:
we may want to consider a constant for default field oriented or not.

Reply:
yea their r still a lot of things that should be constants and are not currently